### ISSUE TITLE
Login Application Error

### ISSUE THREAD

**Issue by btrnew** [spawaskar-cora/cora-docs#255]
When I attempted to login this morning I received this application error. I screenshotted it, went back to the login page, tried again, and got in just fine. Not sure why this error popped up, but I figured I should report it. 

<img width="960" alt="cora error" src="https://user-images.githubusercontent.com/43450732/45836988-f6bee280-bca9-11e8-9b62-a6e03c9f0df9.png">

**Comment by emilystreetman** [comment]
I had the same issue a couple of times today (in cora-cat, though) - I was not able to figure out why it worked vs. didn't either.

**Comment by btrnew** [comment]
Yes, as a follow up, I tried to get into cora-cat this morning and all I am getting is the application error. I've refreshed, closed browser, etc. probably 20 different times with no luck.

**Comment by tlvandeest** [comment]
I have also been getting this error periodically all day when trying to get to different menu options (e.g., Review page, Measurement page) or Searching for an element within CoRA Cat.

**Comment by SachinPawaskarUNO** [comment]
@fedamann, My research on the application issues that users are seeing is due to memory issues related to sizing of the server. We need to have a conversation on getting more resources for the rollout of the next release.

I will continue to dig deeper to identify optimizations.

![image](https://user-images.githubusercontent.com/19599773/46067373-48e58500-c13c-11e8-9bea-f2cafe95bf6a.png)

**Comment by emilystreetman** [comment]
Should we close this ticket if no one sees the problem between now and the next scrum?

**Comment by emilystreetman** [comment]
Per scrum 9OCT18, this ticket will be left open for a few weeks.
